src/ODBCPdoStatement.php

```diff
@@ -60,6 +53,14 @@ public function fetchAll($how = NULL, $class_name = NULL, $ctor_args = NULL)
     {
         $records = [];
         while ($record = $this->fetch()) {
+            for ($i=1; $i <= odbc_num_fields($this->statement); $i++) { 
+                $fieldName = odbc_field_name($this->statement , $i);
+                if(odbc_field_type($this->statement , $i)=='DECIMAL') {

```

Comment:
why converting a field from decimal to integer and not float?

Based on the Snowflake documentation, I'd think that `DECIMAL` is a float
https://docs.snowflake.com/en/sql-reference/data-types-numeric.html
And if we return a DECIMAL for an ID, maybe we need to change the data type in our model.

Reply:
If I understood the documentation correctly, in snowflake all numeric types are `Number` and seems the odbc driver will always convert to `DECIMAL` (https://community.snowflake.com/s/question/0D50Z000092f9Z8SAI/is-there-a-fix-to-odbc-datatable-column-that-returns-decimal-instead-of-int). 

I checked that `odbc_field_type` returns `DECIMAL` to all `Number` type coming from snowflake.

You have a point, I'll change the `intval` to `floatval`.